src/index.js

```diff
@@ -85,7 +85,9 @@ export default class WorkerPlugin {
             return ParserHelpers.addParsedVariableToModule(parser, id, req);
           };
           
-          parser.hooks.new.for('Worker').tap(NAME, handleWorker('Worker'));
+          if (this.options.worker === undefined || this.options.worker) {
```

Comment:
```suggestion
          if (this.options.worker !== false) {
```

Reply:
Okay with this change so long as we're also okay that a "falsey" option like `{ worker: null }`  will not disable bundling of `Worker`